packages/iguazu-holocron/package.json

```diff
@@ -43,9 +43,9 @@
     "babel-plugin-add-module-exports": "^0.2.1",
     "enzyme": "^3.1.0",
     "enzyme-to-json": "^3.1.2",
+    "react": "^16.0.0",
```

Comment:
Each package have different versions of react. Is it intended?

Reply:
I kept the versions each package already had just in case they were using a feature not available until that version. They are different because they were written at different times, so when the writer did the initial install the latest version was different.